cinema/urls.py

```diff
@@ -1 +1,26 @@
-# write urls here
+from django.urls import include, path
+from rest_framework.routers import DefaultRouter
+from .views import (
+    GenreViewSet,
+    ActorViewSet,
+    CinemaHallViewSet,
+    MovieViewSet,
+    MovieSessionViewSet,
+    OrderViewSet,
+    TicketViewSet
+)
```

Comment:
Imports should be grouped and separated by a blank line according to standard library, external libraries, and custom imports. This practice enhances both code readability and maintainability. 
Read more about topic [here](https://peps.python.org/pep-0008/#imports)